src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAgreement/GetOrganisationsToRemove.cshtml

```diff
@@ -25,32 +25,28 @@
                     <th scope="col"><span class="vh">Status</span></th>
             </thead>
             <tbody>
+                @{
```

Comment:
Can we change this code to check the Agreements Collection for any orgs that can be removed rather than this method of setting a count?